src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}: PaginationProps) => {
+  const pagiPagesCount = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1
+            ? 'true'
+            : 'false'
+          }
+          onClick={() => currentPage > 1
+            ? onPageChange(currentPage - 1)
+            : ''
+          }
```

Comment:
it's better to move this logic to an own function
```
const handlePrevClick = () => {
  if (currentPage > 1) {
    onPageChange(currentPage - 1)
  }
}
```
```suggestion
          onClick={handlePrevClick}
```